adapters/marsmedia/marsmedia_test.go

```diff
@@ -0,0 +1,11 @@
+package marsmedia
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapterstest"
```

Comment:
Should be `github.com/prebid/prebid-server/adapters/adapterstest`. This is currently causing the Travis CI error.

Reply:
I am seeing this error:
marsmediatest/exemplary/simple-banner.json: MakeRequests had wrong error count. Expected 0, got 1
What can couse to this error?

Reply:
In this case, it's because the `simple-banner.json` test file has `ext.marsmedia.zone` instead of the expected `ext.bidder.zone`.

Reply:
Thanks you very match for all.
The test running without errors.